iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/SlackTerminalNodeBreakerImpl.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.iidm.network.Terminal;
+import com.powsybl.iidm.network.VoltageLevel;
+
+/**
+ * @author Florian Dupuy <florian.dupuy at rte-france.com>
+ */
+public class SlackTerminalNodeBreakerImpl extends AbstractExtension<VoltageLevel> implements SlackTerminal {
```

Comment:
This extension should inherit from `AbstractMultiVariantConnectableExtension` because the slack bus could depend from the variant.